src/navigations/root-stack.js

```diff
@@ -3,67 +3,68 @@ import * as React from 'react';
 import CreatePost from '../screens/CreatePost';
 import DiscoveryScreenV2 from '../screens/DiscoveryScreenV2';
 import DomainScreen from '../screens/DomainScreen';
 import FollowersScreen from '../screens/Followings/FollowersScreen';
 import FollowingScreen from '../screens/Followings/FollowingScreen';
+import HelpCenter from '../screens/WebView/HelpCenter';
+import HomeBottomTabs from './HomeBottomTabs';
 import ImageViewerScreen from '../screens/ImageViewer';
-import ChooseUsername from '../screens/InputUsername';
+import KeyboardWrapper from './KeyboardWrapper';
 import LinkContextScreen from '../screens/LinkContextScreen';
 import LocalCommunity from '../screens/LocalCommunity';
+import NetworkStatusIndicator from '../components/NetworkStatusIndicator';
+import OneSignalNavigator from './OneSignalNavigator';
 import OtherProfile from '../screens/OtherProfile';
 import OtherProfilePostDetail from '../screens/OtherProfilePostDetail';
 import OtherProfileReplyComment from '../screens/OtherProfileReplyComment';
 import PostDetailPage from '../screens/PostPageDetail';
+import PrivacyPolicies from '../screens/WebView/PrivacyPolicies';
 import ProfilePostDetail from '../screens/ProfilePostDetail';
 import ProfileReplyComment from '../screens/ProfileReplyComment';
 import ReplyComment from '../screens/ReplyComment';
 import Settings from '../screens/Settings';
 import SignIn from '../screens/SignInV2';
+import SignedChatScreen from '../screens/ChatScreen/SignedChatScreen';
 import TopicMemberScreen from '../screens/TopicMemberScreen';
 import TopicPageScreen from '../screens/TopicPageScreen';
 import Topics from '../screens/Topics';
 import VideoViewerScreen from '../screens/VideoViewer';
-import HelpCenter from '../screens/WebView/HelpCenter';
-import PrivacyPolicies from '../screens/WebView/PrivacyPolicies';
 import WhotoFollow from '../screens/WhotoFollow';
 import api from '../service/config';
+import useLocalDatabaseHook from '../database/hooks/useLocalDatabaseHook';
+import {
+  AddParticipant,
+  ChannelScreen,
+  ChatDetailPage,
+  ContactScreen,
+  DetailDomainScreen,
+  DetailGroupImage,
+  GroupInfo,
+  GroupMedia,
+  GroupSetting,
+  ProfileScreen
+} from '../screens';
 import {InitialStartupAtom, LoadingStartupContext} from '../service/initialStartup';
 import {NavigationConstants} from '../utils/constants';
-import HomeBottomTabs from './HomeBottomTabs';
-import KeyboardWrapper from './KeyboardWrapper';
-import OneSignalNavigator from './OneSignalNavigator';
+import {followersOrFollowingAtom} from '../screens/ChannelListScreen/model/followersOrFollowingAtom';
+import {useInitialStartup} from '../hooks/useInitialStartup';
 
 const RootStack = createNativeStackNavigator();
 
 export const RootNavigator = () => {
   const initialStartup = useRecoilValue(InitialStartupAtom);
   const [following, setFollowing] = useRecoilState(followersOrFollowingAtom);
   const loadingStartup = useInitialStartup();
+  useLocalDatabaseHook(true);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useLocalDatabaseHook` is being called with a boolean argument, but it's not clear what this argument represents. It would be better to use an options object for clarity and future extensibility.
```diff
-   useLocalDatabaseHook(true);
+   useLocalDatabaseHook({withMigration: true});
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->